src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Node<K, V> {
+    protected final int hash;
+    protected final K key;
+    protected V value;
+    protected Node<K, V> next;
+
+    public Node(int hash, K key, V value, Node<K, V> next) {
+        this.hash = hash;
+        this.key = key;
+        this.value = value;
+        this.next = next;
+    }
+}
```

Comment:
why did you use Nodes as a separate class, you can use it as an inner class of MyHashMap and apply private access modifiers to fields

Reply:
totally agree, moved it to the class of HashMap. Thanks!